docs/diagrams/DeleteCommandSequenceDiagram.puml

```diff
@@ -0,0 +1,30 @@
+@startuml
+actor user
+
+user -> ":Ui" ++ : "delete n/John i/1"
+create ":Parser"
+":Ui" -> ":Parser" ++: new Parser("delete n/John i/1")
+":Parser" --> ":Ui" --: parser:Parser
+":Ui" -> ":Parser" ++: parseCommand("delete n/John i/1")
+create ":DeleteCommand"
+":Parser" -> ":DeleteCommand" ++: new DeleteCommand("John": String,1:int)
+":DeleteCommand" --> ":Parser" --: deleteCommand:DeleteCommand
+":Parser" --> ":Ui" --: deleteCommand:DeleteCommand
+":Ui" -> ":DeleteCommand" ++: execute(masterTimetable)
+":DeleteCommand" -> ":MasterTimetable" ++: getByName("John")
+":MasterTimetable" --> ":DeleteCommand" ++ : timetable: Timetable
+":DeleteCommand" -> ":Timetable" ++: get(index - 1)
+":Timetable" --> ":DeleteCommand" -- : event: Event
+alt event is type meeting
+":MasterTimetable" -> ":MasterTimetable" ++: deleteMeetingFromTimetable(timetable:Timetable,meeting:Meeting)
+":MasterTimetable" -> ":MasterTimetable" --: deleteMeetingFromMeetingList(meeting:Meeting)
+":MasterTimetable" -> ":DeleteCommand" --: deleteMeetingFromAllTimetableConfirmation(meeting:Meeting)
```

Comment:
Again, maybe consider describing these calls in words instead of writing the entire function call, so that the diagram is less stretched horizontally and more readable?